lib/ansible/inventory/manager.py

```diff
@@ -161,13 +165,59 @@ def __init__(self, loader, sources=None, parse=True, cache=True):
         else:
             self._sources = sources
 
+        self._launch_ssh_agent()
```

Comment:
I think this belongs does in prefork but not this early, move to strategy?  (really in a connection manager as part or execution engine) as inventory does not require connection to the hosts at all. It can provide the info but should not be in charge of setting up communications or their prereqs.  The good thing is that this code is easy to move later on, so I do not think it is a blocker for the PR right now.